.vscode/launch.json

```diff
@@ -0,0 +1,37 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "vcpkg-ce cli",
+      "request": "launch",
+      "type": "node",
+      "program": "${workspaceRoot}/ce/ce/dist/main.js",
+      "console": "integratedTerminal",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "outFiles": [
+        "${workspaceFolder}/ce/ce/*/dist/**/*.js",
+        "!**/node_modules/**"
+      ]
+    },
+    {
+      "name": "MochaTest",
+      "type": "pwa-node",
+      "request": "attach",
+      "port": 9229,
+      "continueOnAttach": true,
+      "autoAttachChildProcesses": false,
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "outFiles": [
+        "${workspaceFolder}/ce/*/dist/**/*.js",
+        "!**/node_modules/**"
+      ]
+    }
+  ]
+}
```

Comment:
```suggestion
```
also the other .vscode stuff

Reply:
And add to `.gitignore`.

Reply:
These are already in .gitignore, that's why they were missed before. They are necessary for any development on the ecmascript stuff to work; otherwise VS Code can't find the test harness.

Reply:
I'm confused; is the suggesting asking to delete the file entirely?

Reply:
Nicole and I resolved this in person to ensure Mocha Test Explorer in VS Code still works but stripped everything else